clients/python/text_generation/client.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         headers: Optional[Dict[str, str]] = None,
         cookies: Optional[Dict[str, str]] = None,
         timeout: int = 10,
+        cert: Optional[Union[str, tuple[str, str]]] = None,
```

Comment:
Could you use Tuple from typing instead of tuple so it works with older version of python? This will cause an error on import with python <3.9.